tools/releases/version.sh

```diff
@@ -1,30 +1,52 @@
 #!/usr/bin/env bash
 
-# Kuma version is built as follows:
+# Kuma version info has the following format:
```

Comment:
Following Charlys advice I moved everything version related to this file so that we don't do this in a makefile, but now we have to do this weird mapping in https://github.com/kumahq/kuma/pull/5291/files#diff-f86fd3b59b066ba96f4a8b51a82c94d1113a7090a87b3bb9635830bdcbf1e8fdR11.